### DISCUSSION TITLE
界面"数量"描述的建议，用过之后才知道每个数量描述的意义，建议改为以下，更直观。

### DISCUSSION THREAD

**Discussion by piaoliuchong** [Guovin/iptv-api#770] [General]
频道接口数量       ： 输出结果数量
ipv4数量         ：   参与筛选数量
ipv6数量         ：  参与筛选数量
结果来源数量   ：输入列表数量

**Comment by Guovin** [comment]
所有的数量定义都是基于结果的，你理解错了。
